sky/skylet/providers/gcp/config.py

```diff
@@ -506,10 +507,50 @@ def _configure_subnet(config, compute):
     ):
         return config
 
-    subnets = _list_subnets(config, compute)
+    subnets = _list_subnets(config, compute, filter=f'(name="{SKYPILOT_VPC_NAME}")')
 
     if not subnets:
-        raise NotImplementedError("Should be able to create subnet.")
+        # Create a default VPC network and firewall rules
```

Comment:
```suggestion
        # SkyPilot: Create a default VPC network and firewall rules
```

Reply:
Added before we call `get_usable_vpc(config)`